atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -94,6 +95,23 @@ private AtlasDbHttpClients() {
                 .target(failoverFeignTarget);
     }
 
+    public static <T> T createProxyWithQuickFailover(
```

Comment:
I'd suggest you can name this as `createProxyWithFailover()` (still with the 5 arguments), and have the above factory call it. The `3000` is a constant from the `FailoverFeignTarget` that perhaps could be made public.

The 4th argument could be an `Optional<Request.Options>` maybe even - then we could apply options to the builder only if the optional is present, perhaps?

Then you'd have  `createProxyWithFailover(sslSocketFactory, endpointUris, 3000, Optional.absent(), type);` in the other method. (There might be an even better way to do this too!)